lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -169,4 +172,19 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
       ie -> if InformedEntity.parent_station?(ie), do: ie.stop
     end)
   end
+
+  defp has_nearby_redundancy?(
+         %Alert{informed_entities: [%{facility: %{id: informed_facility_id}} | _]},
+         redundancy_data
+       ) do
+    Enum.find(redundancy_data, &(&1.id == informed_facility_id))[:nearby_redundancy?]
+  end
+
+  defp fetch_elevator_redundancy_data do
+    :screens
+    |> :code.priv_dir()
+    |> Path.join("elevators/elevator_redundancy_data.json")
+    |> File.read!()
+    |> Jason.decode!(keys: :atoms)
```

Comment:
We could do this once at compile time, instead of re-reading and decoding the file on every elevator screen request, by assigning a module attribute instead. As a bonus the app will then fail to compile if we get the path wrong or something, instead of crashing at runtime.